playbooks/pulp.yml

```diff
@@ -6,6 +6,7 @@
     - operator_sdk.util
   vars:
     default_settings:
+      api_root: '/pulp/'
```

Comment:
is there any sort of validation in case this variable will not have expected slashes around?

Reply:
I've added it now